README.md

```diff
@@ -30,18 +30,7 @@ npm run setup:git-hooks
 ```
 
 **Linting** <br>
-For linting, use your favorite code editor to enable tslint.
-If you use vscode you can change your workspace settings as following
-
-```
-{
-  "eslint.enable": false,
-  "jshint.enable": false,
-  "tslint.enable": true,
-  "tslint.jsEnable": false,
-  "tslint.run": "onType"
-}
-```
+To see linting errors in your editor, you should install an ESLint extension
```

Comment:
Glad you changed this file too! Could we update this sentence a bit. Perhaps we could update this to - 
"ESLint is used for project linting. We suggest installing the ESLint extension in your preferred code editor." 
Up to your discretion there.

Reply:
Good suggestion! I'll be a bit more explicit here.